api/logic/modplatform/atlauncher/ATLPackInstallTask.cpp

```diff
@@ -108,8 +108,12 @@ QString PackInstallTask::getDirForModType(ModType type, QString raw)
         case ModType::Decomp:
             qWarning() << "Unsupported mod type: " + raw;
             return Q_NULLPTR;
+        case ModType::TexturePack:
+            return "texturepackss";
```

Comment:
is this a typo?

Reply:
it is, good spot!

Reply:
Resolved by https://github.com/MultiMC/MultiMC5/pull/3314/commits/44a12a9836eb692f2ad23ca03a016bb67b6d616f.